server/routerlicious/packages/routerlicious/config/config.json

```diff
@@ -196,7 +196,8 @@
         "groupId": "scriptorium",
         "checkpointBatchSize": 1,
         "checkpointTimeIntervalMsec": 1000,
-        "enableRunWithRetryMetricTelemetry": true
+        "enableRunWithRetryMetricTelemetry": true,
+        "enableHandlerTelemetry": true
```

Comment:
Would it be possible to share functionality between these 2 settings? `enableRunWithRetryMetricTelemetry` and `enableHandlerTelemetry`? Both of them have a similar goal, which is to control telemetry in Scriptorium. If it makes sense, using a single setting for both of them might help with increasing our already long list of settings.

Reply:
I think that would make sense. Updated to have a single flag for both.